src/documents/CoreAPI.md

```diff
@@ -539,6 +539,22 @@ if you are interested.
 path segments representing this
 * `newValue: {Any}` The value to be written into `model` at the path `path`
 
+### fluid.modelPairToChanges(value, oldValue, changePathPrefix)
+
+* `value {Any}` Model value to compare
+* `oldValue {Any}` Model value to compare
+* `changePathPrefix {String|Array of String}` [optional] Path prefix to prepend to change record paths
+* Returns: `{Array of Object}` An array of change records
```

Comment:
I think I was incorrect in referring to the output as "change records". Looking at the ChangeApplierAPI docs, I think they are changeRequest objects, because I output "path" properties rather than "changePath" properties. Does this seem right?

I've updated the docs to read "changeRequest" and added a link to the docs for it.